playground/orleans/OrleansServiceDefaults/Extensions.cs

```diff
@@ -60,8 +65,9 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
                 tracing.AddSource("Microsoft.Orleans.Application");
 
                 tracing.AddAspNetCoreInstrumentation()
-                       .AddGrpcClientInstrumentation()
-                       .AddHttpClientInstrumentation();
+                    // Uncomment the following line to enable gRPC instrumentation (requires the OpenTelemetry.Instrumentation.GrpcNetClient package)
+                    //.AddGrpcClientInstrumentation()
```

Comment:
Since the TestShop does in fact use Grpc, can we keep this one? That way we get the right tracing in the TestShop.